static/js/unread.js

```diff
@@ -413,6 +413,22 @@ exports.update_unread_topics = function (msg, event) {
     );
 };
 
+exports.update_unread_moved_topic = function (msg) {
+    if (!unread_messages.has(msg.id)) {
+        return;
+    }
+
+    exports.unread_topic_counter.delete(
+        msg.id
+    );
+
+    exports.unread_topic_counter.add(
+        msg.stream_id,
+        msg.topic,
+        msg.id
+    );
+};
+
```

Comment:
Did you test the unread behavior after using `manage.py mark_all_message_unread` moves the unread counts correctly?

Reply:
No, let me try.